cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -64,6 +64,8 @@ API Changes
 Known Issues
 ------------
 
+- CDAP has been tested on and supports CDH 4.2.x through CDH 5.3.x. It has not been tested on later versions.
```

Comment:
Sure... yeah that's correct.  Except for the lower bound, I don't think ``0.94.2.x`` is the correct nomenclature, since the versions we support would be ``0.94.2``, ``0.94.3``, and so on...  ``0.94.x where x >= 2`` is a verbose but correct description :)  

FYI ``0.94.2`` is the version of Hbase included in CDH 4.2.0, which is why it's our minimum.